docs/source/installation.mdx

```diff
@@ -131,6 +131,24 @@ Tags: ['pytorch', 'tf', 'jax', 'tflite', 'rust', 'safetensors', 'gpt2', 'text-ge
 Task: text-generation
 ```
 
+## Windows limitations
+
+With our goal of democratizing good ML everywhere, we built `huggingface_hub` to be a
+cross-platform library and in particular to work correctly on both Unix-based and Windows
+systems. However, there are a few cases where `huggingface_hub` has some limitations when
+run on Windows. Here is an exhaustive list of known issues. Please let us know if you
+encounter any undocumented problem by opening [an issue on Github](https://github.com/huggingface/huggingface_hub/issues/new/choose).
+
+- `huggingface_hub`'s cache system relies on symlinks to efficiently cache files downloaded
+from the Hub. On Windows, you must activate developer mode or run your script as admin to
+enable symlinks. Please read [this page](./how-to-cache#limitations) for more details.
```

Comment:
Maybe mention here that if this isn't activated, it will still work, just in a non-optimized manner

Reply:
Good idea !